libsol/system_instruction.c

```diff
@@ -389,7 +389,7 @@ static int print_system_authorize_nonce_info(
     SummaryItem* item;
```

Comment:
On 351, `Advance Nonce` is the only system summary_primary_item in Title Case. (Commenting here, as github won't let me comment back that far)

Reply:
D'oh!  Missed one.  Nice catch!